app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -1,6 +1,11 @@
 module ManageIQ::Providers::EmbeddedAnsible::CrudCommon
   extend ActiveSupport::Concern
 
+  def self.included(model)
+    model.include SupportsFeatureMixin
+    model.supports :create
+  end
```

Comment:
We are using ActiveSupport::Concern, so this simplifies to:

```suggestion
  included do
    include SupportsFeatureMixin
    supports :create
  end
```

Reply:
I kinda like just using ruby, but I guess this works too